pytorch_lightning/core/optimizer.py

```diff
@@ -149,6 +149,7 @@ def closure_dis():
                 with opt_dis.toggle_model(sync_grad=accumulated_grad_batches):
                     opt_dis.step(closure=closure_dis)
         """
+        # is separating optimizer_step_without_closure and optimizer_step_with_closure still needed?
```

Comment:
question to reviewers

Reply:
This is only relevant for manual optimization. It can be helpful information if the manual closure is particularly expensive.

Reply:
remember to remove the line before resolving